synapse/rest/client/v2_alpha/account.py

```diff
@@ -295,7 +296,7 @@ def on_GET(self, request, medium):
             )
             request.setResponseCode(e.code)
 
-        request.write(html.encode('utf-8'))
+        request.write(ensure_binary(html))
```

Comment:
I'm not a fan of this solution. variables should be either be bytes or unicode, irrespective of whether we are on python 2 or 3, and where we need to convert between them there should be explicit conversions. So a bytes thing should be `bytes` on python3 and `str` on python2, and a unicode thing should be `str` on python3 and `unicode` on python2.

so is `load_jinja2_template` returning a unicode value (as `str`/`unicode`), or something that has been encoded in utf-8 (as `bytes`/`str`)? If the answer is "it depends", the bug is in `load_jinja2_template`, not here